### PR TITLE
header search

### PR DESCRIPTION
**Pull request by Lamperoyge**
https://www.loom.com/share/61f40c93a11b4d9693ce640e380cedaa

global search

### CODE REVIEW

**Comment by terryli0095** [comment]
this is good, although i thought
 we wanted the header search to be a more global search

**Comment by Lamperoyge** [comment]
> this is good, although i thought we wanted the header search to be a more global search

oh, I think I can do that, I can create an endpoint for this

**Comment by andywong418** [comment]
Is there a backend PR for this @Lamperoyge

**Comment by Lamperoyge** [comment]
> Is there a backend PR for this @Lamperoyge

yep, it's already merged https://github.com/wondrous-dev/wondrous-backend/pull/498

File: wondrous-app/components/CreateEntity/CreateEntityModal/index.tsx [hunk 1]
```diff
@@ -1326,7 +1326,7 @@ export const CreateEntityModal = (props: ICreateEntityModal) => {
   const getPaymentMethodData = (id) => paymentMethods.find((payment) => payment.id === id);
 
   const handleSaveTemplate = (template_name) => {
-    const rewards = _.isEmpty(form.values.rewards)
+    const rewards = isEmpty(form.values.rewards)
```
Comment (terryli0095) [inline wondrous-app/components/CreateEntity/CreateEntityModal/index.tsx:1329]: nice

**Pull request merged**